contact.py

```diff
@@ -6,22 +6,30 @@ def __init__(self):
         self.db = []
         self.id = 0
```

Comment:
What does this do? Why would a contact database manager need an id?

Reply:
we made this one to use it instead of list index (when deleting an entry). right?

Reply:
Ohhhh yeah let's rename this to something more descriptive like `id_sequence_counter`.